config/configuration_singleton.rb

```diff
@@ -96,6 +96,16 @@ def production_database_path
     # FIXME: add support/handling for DATABASE_URL
     Pathname.new(ENV["DATABASE_PATH"] || dataroot.join('production.sqlite3')).expand_path
   end
+
+  def database_path
+    {
+      'test' => Pathname.new('db/test.sqlite3'),
+      'development' => Pathname.new('db/development.sqlite3')
+    }.fetch(
+      rails_env,
+      production_database_path
+    )
+  end
```

Comment:
Did you add this for easy testing of your change in development?

Reply:
The `Hash#fetch`? I wanted `database_path` to be valid for all environments. Ironically using the production value as the default seems to be the safest choice since if the app were installed to `/var/www/ood/...` and it were incorrectly placed into development mode then it would attempt to create the database in a protected directory, but the production database is built in `$HOME`.